code-challenges.test.js

```diff
@@ -10,45 +12,105 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
+describe("arrayMixUpColors", () => {
+  it("returns an array with the first index removed and the other values within the array mixed up", () => {
+    var colors1 = ["purple", "blue", "green", "yellow", "pink"];
+    // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+    var colors2 = [
+      "chartreuse",
+      "indigo",
+      "periwinkle",
+      "ochre",
+      "aquamarine",
+      "saffron",
+    ];
+    // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+    // Use expect method to take in variable and use expect.arrayContaining() for test
+    expect(arrayMixUpColors(colors1)).toEqual(
+      // I kept getting the
+      expect.arrayContaining(["yellow", "blue", "pink", "green"])
+    ),
+      expect(arrayMixUpColors(colors2)).toEqual(
+        expect.arrayContaining([
+          "saffron",
+          "aquamarine",
+          "periwinkle",
+          "indigo",
+          "ochre",
+        ])
+      );
+  });
+});
+// Red - Failed (not defined)
 
 // b) Create the function that makes the test pass.
-
-
+// Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+// Use .shift(), to remove the first item from the array
+const arrayMixUpColors = (array) => {
+  array.shift();
+  // Use .sort(), to sort the items within the array in ascending order
+  let newArr = array.sort((a, b) => a.sort - b.sort);
+  // Iterate over array using .map()
+  array.map(({ value }) => value);
+  return newArr;
+};
+// Green - Passed
```

Comment:
The goal of this prompt is to have the array return the colors in a random order. So yes, ascending order will get your test passing but the outcome will always be the same order. I am also curious about line 59. It looks like you are mapping but not changing the array at all.